server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -400,7 +400,6 @@ public class ShuffleServerConf extends RssBaseConf {
   public static final ConfigOption<Integer> NETTY_SERVER_PORT = ConfigOptions
       .key("rss.server.netty.port")
       .intType()
-      .checkValue(ConfigUtils.POSITIVE_INTEGER_VALIDATOR_2, "netty port must be positive")
```

Comment:
Why do we remove this line?

Reply:
You can see https://github.com/apache/incubator-uniffle/pull/739#discussion_r1141031976 and  https://github.com/apache/incubator-uniffle/pull/739#discussion_r1141031669

Reply:
I am also wondering if it is possible to reserve a negative number of ports, which means shutting down the Netty service.